### PR TITLE
Fix: Remove inheritance error due to missing function in NeptuneGremlinProxy

### PR DESCRIPTION
**Pull request by rajat499**
### Summary of Changes
```NeptuneGremlinProxy``` inherits ```AbstractGremlinProxy``` class which doesn't contain ```get_popular_resources()``` function. Hence the following error is produced:
```
  File "/app/metadata_service/proxy/__init__.py", line 47, in get_proxy_client
    client_kwargs=client_kwargs)
TypeError: Can't instantiate abstract class NeptuneGremlinProxy with abstract methods get_popular_resources
```

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes.
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does

### CODE REVIEW

**Comment by verdan** [comment]
@rajat499 we already have a PR to fix this: https://github.com/amundsen-io/amundsen/pull/1289

File: metadata/metadata_service/proxy/gremlin_proxy.py [hunk 1]
```diff
@@ -1773,6 +1773,12 @@ def get_resource_generation_code(self, *,
                                      uri: str,
                                      resource_type: ResourceType) -> GenerationCode:
         pass
+    
+    def get_popular_resources(self, *,
+                              num_entries: int,
+                              resource_types: List[str],
+                              user_id: Optional[str] = None) -> Dict[str, List]:
+        pass
```
Comment (verdan) [inline metadata/metadata_service/proxy/gremlin_proxy.py:1781]: Can you please make sure to raise the `NotImplementedError` instead? 

ref: https://github.com/amundsen-io/amundsen/pull/1289/files

**Review by verdan — CHANGES_REQUESTED** [review]
one comment.

File: metadata/metadata_service/proxy/gremlin_proxy.py (hunk 1 above)
Comment (rajat499) [reply metadata/metadata_service/proxy/gremlin_proxy.py]: @verdan Done.

**Comment by verdan** [comment]
@rajat499 seems like you also need to fix the build.

**Comment by rajat499** [comment]
> @rajat499 seems like you also need to fix the build.

@verdan Yes. Fixed it.

**Review by verdan — APPROVED** [review]

**Pull request merged**